### DISCUSSION TITLE
How do we start multiple plugins at once?

### DISCUSSION THREAD

**Discussion by proudhon** [jarun/nnn#1954] [Q&A] [answered]
After defining plugins keys:

`export NNN_PLUG='p:preview-tui;0:.ntfy'`

how can we start a list of them?

Neither of these work:

`nnn -P p 0`
`nnn -P p -P 0`

**Comment by N-R-K** [comment, marked as answer]
Not possible. Only a single plugin can be started with `-P`.

**Comment by jarun** [comment]
If you need to run a few plugins at the start, have a master plugin that calls the other scripts one by one.
